Explorer/Assets/DCL/Minimap/MinimapController.cs

```diff
@@ -154,6 +153,7 @@ private void QueryPlayerPosition(in CharacterTransform transformComponent)
         {
             Vector3 position = transformComponent.Position;
 
+            // TODO: this is always null, why keep this branch?

```

Comment:
It's not always null, query executes every frame and lazily intiializes `mapCameraController`. If it is already instantiated it just set position via `mapRendererTrackPlayerPosition`

Reply:
True